packages/thumbprint-react/components/Fab/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
```

Comment:
We've used `noop` in some other components but have been trying to move away from it since it isn't providing anything helpful. (Plus, it makes testing a bit harder since, if I remember correctly, `noop` is truthy.)